src/dataflow/src/render/context.rs

```diff
@@ -206,58 +206,44 @@ where
     /// If `key` is set, this is a promise that `logic` will produce no results on
     /// records for which the key does not evaluate to the value. This is used to
     /// leap directly to exactly those records.
-    pub fn flat_map<I, L>(
+    pub fn flat_map<I, C, L>(
```

Comment:
Doc comments now out of date (`logic` is no longer the thing that takes keys as input, for example).

Reply:
Also, would probably help to indicate what argument `L` takes (I think morally it is a `(&Row, &Row)` though perhaps represented as a `&[&Row]`, so only expecting two entries?

Reply:
Updated the doc!